### ISSUE TITLE
2.x: Customize the filling of LABEL and ORDERLABEL in calendar structures

### ISSUE THREAD

**Issue by matthias-ronge** [kitodo/kitodo-production#2021]
For newspapers as well as seasonal concert programs, the current year (structure element “year”) is always assigned to the `TitleDocMain` metadata, although this is only necessary if it is to have **a different name** than the annual value. For example: For the seasonal concert programs “Season 1852/1853” in the structure element “year” the metadata `TitleDocMainShort` would have to be filled with “1852/1853”, the metadata `TitleDocMain"` however with “Season 1852/1853”. For a season “1830/1831”, ie without an explicit season identifier, `TitleDocMainShort` would have to be assigned as “1830/1831” at the annual level and `TitleDocMain` would be empty. For a newspaper, this should be analogous: For the year 1852, the value “1852” appears in the structure element “year” in the metadata `TitleDocMainShort`, whereas the metadata `TitleDocMain` is empty or not assigned in the structure element `year`.

In summary:
* All calendar entries (i.e. “year”, “month” and “day”) must always have the date in the `ORDERLABEL` aka `TitleDocMainShort`.
* For all calendar entries, `LABEL` must only be used if `TitleDocMain` is allowed(and should then have the same value.
